lib/Sema/CSDiagnostics.h

```diff
@@ -3138,8 +3138,6 @@ class ConcreteTypeSpecialization final : public FailureDiagnostic {
 };
 
 class GenericFunctionSpecialization final : public FailureDiagnostic {
-  ValueDecl *Decl;
```

Comment:
I think you need to remove this from initializer as well?

Reply:
Actually if you don't mind instead of removing this could you add it as a note in `diagnoseAsError`? I think that was the intention behind leaving it around... I can do that too.

Reply:
Ok, I will drop this change from the PR and let you handle it